common/persistence/class.PhpRedisDriver.php

```diff
@@ -189,30 +192,94 @@ public function hGet($key, $field)
 
     public function hDel($key, $field): bool
     {
-        return (bool) $this->callWithRetry('hDel', [$key, $field]);
+        return (bool)$this->callWithRetry('hDel', [$key, $field]);
     }
 
     //Time complexity: O(N) where N is the size of the hash.
     public function hGetAll($key)
     {
         return $this->callWithRetry('hGetAll', [$key]);
     }
+
     //Time complexity: O(N)
     public function keys($pattern)
     {
         return $this->callWithRetry('keys', [$pattern]);
     }
+
     //Time complexity: O(1)
     public function incr($key)
     {
         return $this->callWithRetry('incr', [$key]);
     }
+
     //Time complexity: O(1)
     public function decr($key)
     {
         return $this->callWithRetry('decr', [$key]);
     }
 
+    /**
```

Comment:
question: im not sure whats the scope of this PR :). i can see there are new functions, but shouldnt this functions be specified in a contract interface? and if so, shouldnt the other implementations include this functions to not break when used? (i can see this PR doesnt use this methods anywhere so its safe, just a bit lost :D)

Reply:
Adding to the contract will cause a breaking change and I would have to implement them to all drivers. That is why I did not :)

Reply:
What is your suggestion? That is the problem with interfaces and open source projects, every change in the interface breaks change to everybody (customers and other users that we do not even know).

I am aware it is not the best approach, but do you have an idea to do it without breaking changing and under the budget we have (almost ending)?